quelpa.el

```diff
@@ -263,6 +268,39 @@ and return NEW-STAMP-INFO, otherwise return OLD-STAMP-INFO."
           new-stamp-info)
       old-stamp-info)))
 
+(defun quelpa-check-dir-hash (name config dir build-dir)
+  "Check if hash of DIR contents is different as in STAMP-FILE.
+If it is different save the new hash and timestamp to STAMP-FILE
+and return TIME-STAMP, otherwise return OLD-TIME-STAMP."
+  (let* ((hashes)
```

Comment:
You can use an even shorter form for values bound to `nil`, `(let (foo bar baz) ...)`.  Just drop the parentheses around them.